front/lib/api/assistant/actions/websearch.ts

```diff
@@ -0,0 +1,54 @@
+import type { AgentActionSpecification, Result } from "@dust-tt/types";
+import { Ok } from "@dust-tt/types";
+import type { WebsearchConfigurationType } from "@dust-tt/types/dist/front/assistant/actions/websearch";
+
+import type { BaseActionRunParams } from "@app/lib/api/assistant/actions/types";
+import { BaseActionConfigurationServerRunner } from "@app/lib/api/assistant/actions/types";
+import type { Authenticator } from "@app/lib/auth";
+
+/**
+ * Params generation.
+ */
+
+export class WebsearchConfigurationServerRunner extends BaseActionConfigurationServerRunner<WebsearchConfigurationType> {
+  run(
```

Comment:
`nit` put the `run` function below the `buildSpecification` to align with the other implementations.